docs/cli-tool.md

```diff
@@ -0,0 +1,69 @@
+# CLI Tool
+
+**JVS is not an official Google product.**
+
+[jvsctl](../cmd/jvsctl) facilitates the justification verification flow provided
+by abcxyz/jvs
+
+## Install
+1. Change directory to where jvsctl code lives
+   ```shell
+   cd cmd/jvsctl
+   ```
+
+2. Install jvsctl
+   ```shell
+   go install
```

Comment:
...`go install github.com/abcxyz/jvs/cmd/jvsctl` will be enough. You don't need the first step

Reply:
done